refinery/core/tests.py

```diff
@@ -2295,3 +2296,49 @@ def test_create_workflow_engine_bad_group_name(self):
                 str(galaxy_instance.id),
                 "non-existent group name"
             )
+
+
+class TestIsAbsoluteURL(TestCase):
+
+    def test_is_absolute_url(self):
+        self.assertTrue(is_absolute_url('http://example.org'))
+        self.assertTrue(is_absolute_url('ftp://example.org'))
+
+    def test_is_not_absolute_url(self):
+        self.assertFalse(is_absolute_url('/path/to/file'))
+        self.assertFalse(is_absolute_url('example.org'))
+
+    def test_is_absolute_url_with_empty_input(self):
+        self.assertFalse(is_absolute_url(None))
+        self.assertFalse(is_absolute_url(''))
+
+    def test_get_absolute_url_with_absolute_url(self):
+        self.assertEqual(get_absolute_url('http://example.org'),
+                         'http://example.org')
```

Comment:
Move this test into the next class?

Reply:
Which one is the next class? TestGetAbsoluteURL class is the last one in that file.

Reply:
This is in `TestIsAbsoluteURL` right now: `TestGetAbsoluteURL` begins just below.